pom.xml

```diff
@@ -30,28 +30,28 @@
     </developers>
 
     <properties>
-        <jenkins.version>1.580.1</jenkins.version>
-        <java.level>6</java.level>
+        <jenkins.version>1.642.1</jenkins.version>
```

Comment:
Actually it may be still possible if the plugin does not use the standard build flow (e.g. EWM's Pipeline):
1) Use assumeThat() to check versions in symbol-related tests and just skip them for the old version
2) In Pipeline launch the build twice: against the default core and against the newest core (useful in any case). It can be done by passing jenkins.version property